playwright.config.js

```diff
@@ -0,0 +1,84 @@
+import { defineConfig } from '@playwright/test';
+
+
+// @ts-check
+const { defineConfig, devices } = require('@playwright/test');
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * @see https://playwright.dev/docs/test-configuration
+ */
+module.exports = defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Glob patterns or regular expressions that match test files.*/
+  testMatch: '*todo-tests/*.spec.ts',
```

Comment:
We can remove this line and rename tests as `.spec.js` which playwright will catch by default.
```suggestion
```